tests/conftest.py

```diff
@@ -51,6 +51,7 @@ def pytest_configure():
             "django.contrib.auth.middleware.AuthenticationMiddleware",
         ],
         TOKEN_TIMEOUT=60,
+        Q_CLUSTER={"sync": True},
```

Comment:
What does this do?

Reply:
It's a configuration setting for Django Q that tells it to run things synchronously. Needed for testing so that it doesn't try and talk to an actual job consumer.